### ISSUE TITLE
Bug in calc_returns function

### ISSUE THREAD

**Issue by vdesai2014** [mahyaret/kuka_rl#4]
I believe that line 277 has a bug: 

returns_current = values[-1].detach()

should instead say 

returns_current = rewards[-1].detach()

**Comment by vdesai2014** [comment]
I was wrong
